src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageSearcher.java

```diff
@@ -118,7 +150,7 @@ public void processElementUsages(@NotNull PsiElement element, @NotNull Processor
             LSPExternalReferencesFinder.processExternalReferences(
                     file,
                     element.getTextOffset(),
-                    options.searchScope,
+                    searchScope,

```

Comment:
External references were already using the search scope, but it was extracted into a variable.